bbconf/config_parser/bedbaseconfig.py

```diff
@@ -225,21 +238,27 @@ def _init_qdrant_backend(self) -> QdrantBackend:
                 f"error in Connection to qdrant! skipping... Error: {err}", UserWarning
             )
 
-    def _init_qdrant_text_backend(self) -> QdrantBackend:
+    def _init_qdrant_text_backend(self) -> Union[QdrantBackend, None]:
         """
         Create qdrant client text embedding object using credentials provided in config file
 
         :return: QdrantClient
         """
+
+        _LOGGER.info(f"Initializing qdrant text engine...")
         try:
             return QdrantBackend(
                 dim=TEXT_EMBEDDING_DIMENSION,
                 collection=self.config.qdrant.text_collection,
                 qdrant_host=self.config.qdrant.host,
                 qdrant_api_key=self.config.qdrant.api_key,
             )
-        except Exception as e:
-            _LOGGER.error(f"Error while connecting to qdrant text engine: {e}")
+        except Exception as _:
```

Comment:
Shouldn't we communicate the exact error to the user? Why did we remove it?

Reply:
You are right